src/tools/pub/dependencies.md

```diff
@@ -16,13 +16,14 @@ At the end is a list of
 
 ## Overview
 
-For each dependency, you specify the *name* of the package you depend on
-and the *range of versions* of that package that you allow.
-You can also specify the
-[*source*](/tools/pub/glossary#source),
-which tells pub how to locate the package.
+For each dependency, you specify the _name_ of the package you depend on
+and the _range of versions_ of that package that you allow.
+You can also specify the [_source_][].
```

Comment:
```suggestion
You can also specify the [_source_].
```

is this not sufficient?

Reply:
It is [sufficient](https://kramdown.gettalong.org/syntax.html#reference-links), but would be inconsistent with other links in the corpus.